File: packages/nx-boot-gradle/src/generators/library/files/groovy/src/test/java/__packageDirectory__/HelloServiceTests.java
package <%= packageName %>;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

import static org.assertj.core.api.Assertions.assertThat;


@SpringBootTest
public class HelloServiceTests {

    @Autowired
    private HelloService helloService;

    @Test
    public void shouldReturnHelloWorld() {
        assertThat(helloService.message()).contains("Hello World");
    }

}

File: packages/internal/generators-files/boot/library/shared/kotlin/src/test/kotlin/__packageDirectory__/HelloServiceTests.kt
package <%= packageName %>

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.AfterAll
import org.junit.jupiter.api.BeforeAll
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest


@SpringBootTest
class HelloServiceTests(@Autowired val helloService: HelloService) {

  @BeforeAll
  fun setup() {
    println(">> Setup")
  }

  @Test
  fun `Should return Hello World`() {
    println(">> Should return Hello World")
    assertThat(helloService.message()).contains("Hello World")
  }


  @AfterAll
  fun teardown() {
    println(">> Tear down")
  }


}
